packages/gatsby/cache-dir/loader.js

```diff
@@ -165,19 +164,14 @@ const queue = {
   empty: () => {
     resourcesCount = Object.create(null)
     resourcesArray = []
-    pages = []
-    pathPrefix = ``
+    pathPrefix = `/`
   },
 
   addPagesArray: newPages => {
-    pages = newPages
-    if (
-      typeof __PREFIX_PATHS__ !== `undefined` &&
-      typeof __PATH_PREFIX__ !== `undefined`
-    ) {
```

Comment:
Removing this `if` seems to make related tests unhappy ( https://travis-ci.org/gatsbyjs/gatsby/jobs/374034245#L1484 ) :(

Reply:
We should reset them to the proper values like `''` and `false` instead of `undefined`, also I think we can drop the workaround for Node v6, do we still support that node version?
I see we still support v6 in the `package.json`, I think we can drop the support for that